dimagi/commcare-hq
# Auto update audio path if translation text is changed

## Summary

https://dimagi-dev.atlassian.net/browse/ICDS-1678
## Feature Flag


## Product Description

On ICDS while uploading bulk app translations check if the translation text is changed then the audio path should automatically be updated.
The custom utility functions are in https://github.com/dimagi/commcare-icds/pull/147

## Safety Assurance

- [X] QA labels are set correctly

The changes are meant to be for ICDS and use Commcare Extensions to split out the custom logic in ICDS repo so other envs should not be affected. 
### Automated test coverage



### QA Plan
On ICDS staging we should upload an App translation file and update some of the fields which have audio configured. On successful upload, the audio path for the changed fields should be updated with a new path.

On staging, we should test that it will have no effect and on updating the translation text the audio path should not be updated.


### Safety story

The changes have been tested locally and they should not affect any other environments except ICDS.

### Rollback instructions
